json/Side_Text.txt

```diff
@@ -228828,1335 +228828,1335 @@
-		"tr_text": "",
+		"tr_text": "...More.",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "7",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "モアが無事で、よかったです。<%br>アネットさんも帰ってきてくれてよかった。",
-		"tr_text": "",
+		"tr_text": "I'm glad you're safe, More.<%br>I'm glad that Annette came back, too.",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "8",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "……これからモアが書く日記が楽しいもの<%br>だったら……わたしは嬉しいです。",
-		"tr_text": "",
+		"tr_text": "...From now on, I want you to have<%br>fun things to write in your diary.",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "9",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "……！",
-		"tr_text": "",
+		"tr_text": "......!",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "10",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "悲しいこともきっと、今まで沢山あって<%br>それも全部この日記に詰まってるんだよね……？",
-		"tr_text": "",
+		"tr_text": "You've seen a lot of sad things, haven't you?<%br>And those things are all there in your diary...",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "11",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "今までの、悲しいこともモアの一部だから……<%br>大事にしてほしいなって思う。",
-		"tr_text": "",
+		"tr_text": "More, those sad things are part of what make<%br>you More... Of course you should cherish them.",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "12",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "それで、これからは……<%br>楽しいことを沢山書き残せたらいいな……！",
-		"tr_text": "",
+		"tr_text": "But from now on...<%br>I want you to write lots of fun things...!",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "13",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "……ジェネ。",
-		"tr_text": "",
+		"tr_text": "...Gene.",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "14",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "できれば、その楽しい思い出の中に<%br>わたしもモアの隣に、いたいなって思う。",
-		"tr_text": "",
+		"tr_text": "And, if I can, I want to be right there<%br>next to you in all those happy memories.",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "15",
 		"jp_name": "モア",
 		"tr_name": "More",
 		"jp_text": "…………うん……！",
-		"tr_text": "",
+		"tr_text": "......Yeah...!",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "16",
 		"jp_name": "ブルーノ",
 		"tr_name": "Bruno",
 		"jp_text": "あのキラキラした空間に入ってく勇気が……<%br>俺には無いなぁ……うん、無い。",
-		"tr_text": "",
+		"tr_text": "You just hate to spoil a beautiful moment like this...<%br>Not interrupting anything, are we?",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "17",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "……！！",
-		"tr_text": "",
+		"tr_text": "......!!",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "18",
 		"jp_name": "ジェネ",
 		"tr_name": "Gene",
 		"jp_text": "アネットさん、ブルーノさん<%br>リーダー！",
-		"tr_text": "",
+		"tr_text": "Annette, Bruno, Leader!",
 		"fileID": 80001
 	},
 	{
 		"eventNo": "19",
 		"jp_name": "アネット",
 		"tr_name": "Annette",
 		"jp_text": "……とか言って、リーダーと私を差し置いて<%br>ズカズカ入って行ってるじゃない。",
-		"tr_text": "",
+		"tr_text": "...Please don't drag Leader and<%br>myself into his vulgar intrusion.",
```

Comment:
> myself into his vulgar intrusion.
should be `myself into your`

Reply:
I figured since Gene was the last one to speak before, and mentioned all three of their names, Annette was going like "please don't lump me in with this guy"